lib/exercism/user_exercise.rb

```diff
@@ -51,6 +51,18 @@ def unarchive!
     update_attributes(archived: false)
   end
 
+  def request_help?
+    request_help
```

Comment:
Although thinking about this some more, those methods probably do need different names.